tensorflow_io/genome/python/ops/genome_ops.py

```diff
@@ -17,6 +17,69 @@
 from __future__ import division
 from __future__ import print_function
 
+import tensorflow as tf
 from tensorflow_io.core.python.ops import core_ops
 
 read_fastq = core_ops.read_fastq
+
+
+
+@tf.function
+def _nucleotide_to_onehot(nucleotide):
+  """Encodes a nucleotide using a one hot encoding."""
+  a_onehot = tf.constant([1, 0, 0, 0])
+  c_onehot = tf.constant([0, 1, 0, 0])
+  g_onehot = tf.constant([0, 0, 1, 0])
+  t_onehot = tf.constant([0, 0, 0, 1])
```

Comment:
I moved these tensor constants here (instead of having them as top level module constants), because they apparently must be created on the same graph as the ReadFastq sequences output. By moving these here, this eliminated the following error:
```
ValueError: Tensor("Const:0", shape=(4,), dtype=int32) must be from the same graph as Tensor("ReadFastq:0", shape=(None,), dtype=string).
```
However this is probably slightly less efficient unless tf does some optimizing under the hood for constants like these that might appear multiple times in a graph. If you know of a better way, let me know, otherwise this should be fine for now.